app/src/test/java/org/gnucash/android/test/unit/export/OfxExporterTest.java

```diff
@@ -53,7 +53,7 @@
 @RunWith(RobolectricTestRunner.class)
 //package is required so that resources can be found in dev mode
 @Config(sdk = 21, shadows = {ShadowCrashlytics.class, ShadowUserVoice.class})
-public class OfxExporterTest extends BookHelperTest {
+public class OfxExporterTest {
```

Comment:
why?

Reply:
It's not using anything from BookHelperTest.